SampleApps/QuickStart-SwiftUI/QuickStart/ViewModels/CallbackViewModel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  CallbackViewModel.swift
+//  QuickStart
+//
+// Copyright (c) 2023 ForgeRock. All rights reserved.
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+import Foundation
+
+class CallbackViewModel: ObservableObject {
```

Comment:
do you need a ObservableObject here ? since this model is not publishing anything